megameklab/src/megameklab/ui/MegaMekLabTabbedUI.java

```diff
@@ -88,6 +91,11 @@ public MegaMekLabTabbedUI(MegaMekLabMainUI... entities) {
         // Enable opening unit and mul files by drag-and-drop
         setTransferHandler(new MMLFileDropTransferHandler(this));
 
+        // If you can think of a way to detect when the user makes a change to the entity, let me know.
```

Comment:
We might be able to get by with a 'dirty' flag in the entity or something, but for now let's see how this works out.